gameengine.js

```diff
@@ -154,22 +149,22 @@ class GameEngine {
       this.arrowRightReleased = null;
       this.arrowUpReleased = null;
       this.arrowDownReleased = null;
-    
+
       this.stats.update(); // remove for production
     }
   }
 
   pause() {
     if (this.isPaused) {
-      hideMessage('pause-message');
+      hideMessage();
```

Comment:
Just pulled and the game doesn't resume after pressing p again, does it on your branch?

Reply:
pull again, I snuck in a fix